topics/chip-seq/tutorials/estrogen-receptor-binding-site-identification/tutorial.md

```diff
@@ -0,0 +1,531 @@
+Now we would like to test the quality of the ChIP-seq preparation, to know if your ChIP-seq samples are more enriched than the control (input) samples.
+
+## Correlation between samples
+
+To assess the similarity between the replicates of the ChIP and the input, respectively, it is a common technique to calculate the correlation of
+read counts on different regions for all different samples.
+We expect that the replicates of the ChIP-seq experiments should be clustered more closely to each other than the replicates of the input sample.
+That is, because the input samples should not have enriched regions included - remember the immuno-precipitation step was skiped during the sample preparation.
+
+To do that, we need at first to catch up for all our samples and re-run the previous steps (quality control and mapping) on each sample.
+To save time, we already did that and we can now work directly on the BAM files of the 8 samples
+
+> ### {% icon hands_on %} Hands-on: Correlation between samples
+>
+> 1. Create a new history
+> 2. Import the 8 BAM files from [Zenodo](https://doi.org/10.5281/zenodo.892432) or from the data library into the history
+>    - [`patient1_ChIP_ER_good_outcome.bam`](https://zenodo.org/record/892432/files/patient1_ChIP_ER_good_outcome.bam)
+>    - [`patient1_input_good_outcome.bam`](https://zenodo.org/record/892432/files/patient1_input_good_outcome.bam)
+>    - [`patient2_ChIP_ER_good_outcome.bam`](https://zenodo.org/record/892432/files/patient2_ChIP_ER_good_outcome.bam)
+>    - [`patient2_input_good_outcome.bam`](https://zenodo.org/record/892432/files/patient2_input_good_outcome.bam)
+>    - [`patient3_ChIP_ER_poor_outcome.bam`](https://zenodo.org/record/892432/files/patient3_ChIP_ER_poor_outcome.bam)
+>    - [`patient3_input_poor_outcome.bam`](https://zenodo.org/record/892432/files/patient3_input_poor_outcome.bam)
+>    - [`patient4_ChIP_ER_poor_outcome.bam`](https://zenodo.org/record/892432/files/patient4_ChIP_ER_poor_outcome.bam)
+>    - [`patient4_input_poor_outcome.bam`](url: https://zenodo.org/record/892432/files/patient4_input_poor_outcome.bam)
+>
+> 3. **multiBamSummary** {% icon tool %} with
+>    - "Sample order matters" to `No`
+>    - "Bam file" to the 8 imported BAM files
+>    - "Choose computation mode" to `Bins`
+>    - "Bin size in bp" to `100`
+>       
+>       This corresponds to the length of the fragments that were sequenced; it is not the read length!
+>
+>    - "Distance between bins" to `500000` (to reduce the computation time for the tutorial)
+>    - "Region of the genome to limit the operation to" to `chr1` (to reduce the computation time for the tutorial)
+>  
+>    This tool splits the reference genome into bins of equal size (*e.g.* 10kb) and counts the number of overlapping reads from each sample.
+>
+> 4. **plotCorrelation** {% icon tool %} with
+>    - "Matrix file from the multiBamSummary tool" to the generated multiBamSummary output
+>    
```

Comment:
Here, I will also like to add some little background.

4. To compute and visualize the sample correlation we use **plotCorrelation** from deepTools. This is a fast process that allows the user to quickly try different color combinations and outputs.